script/testing/junit/traces/update.test

```diff
@@ -155,12 +201,13 @@ INSERT INTO update5 (a) VALUES (1),(2),(3);
 statement ok
 UPDATE update5 SET a=a;
 
-query I nosort
-SELECT a FROM update5;
+query I rowsort
+SELECT a FROM update5 ORDER BY a;
```

Comment:
Both rowsort and order by?

Reply:
I botched a save... Will fix and push the updated version.